recipes-core/aziot-edged/aziot-edged_1.4.4.bb

```diff
@@ -8,7 +8,7 @@ inherit cargo
 # how to get aziot-edged could be as easy as but default to a git checkout:
 # SRC_URI += "crate://crates.io/aziot-edged/0.1.0"
 SRC_URI += "git://github.com/Azure/iotedge;protocol=https;nobranch=1;branch=release/1.4"
```

Comment:
What's the expected behavior when the SRC_URI has both "nobranch=1" and "branch=release/1.4"?  It looks like the aziotd and aziotctl recipes only specify "nobranch=1".  Should we be consistent with those here or should everything be "branch=release/1.4"?